### DISCUSSION TITLE
Get GitHub Education Pack

### DISCUSSION THREAD

**Discussion by lavshashank** [community/community#47311] [GitHub Education]
I just got the email from GitHub in which they mentioned that I'm selected as a Github education global campus and I have access of so many things like github pro, assure, Microsoft product, and so many more. 

Thank you Github.

**Comment by jge162** [comment]
Nice job.

**Comment by samus-aran** [comment]
Congratulations @lavshashank 🎉 

What are you planning to redeem first and what you are going to do with the benefit?
